src/templates/home-page.js

```diff
@@ -43,34 +38,6 @@ export const HomePageTemplate = ({
     Aos.init({ once: true })
   }, [])
 
-  // const [
-  //   liquidityRewardsAPYs,
-  //   areLiquidityRewardsAPYsFetching,
-  // ] = useLiquidityRewardsAPY()
-
-  const [coveragePoolAPY, isCoveragePoolAPYFetching] = useCoveragePoolsAPY()
```

Comment:
Non-blocking:

We could probably also remove `src/hooks/useCoveragePoolsAPY.js` file. Unless we want to keep it just in case?